### DISCUSSION TITLE
Grafana Notification Channel - Cisco Webex Teams

### DISCUSSION THREAD

**Discussion by cagincirikciel** [grafana/grafana#40469] [Alerting]
Hi ,

As we know there are multiple collaborations tool, People want to manage all internal communications in one place. In this direction, we try to integrate Grafana and Cisco Webex but we have faced with integration Issues. 

Could you add support for Webex to Notification Channesl like MS Teams?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
